conformance/tests/gateway-with-attached-routes.go

```diff
@@ -90,6 +90,52 @@ var GatewayWithAttachedRoutes = suite.ConformanceTest{
 
 			kubernetes.GatewayStatusMustHaveListeners(t, s.Client, s.TimeoutConfig, gwNN, listeners)
 		})
+
+		t.Run("Gateway listener should have an accepted http route attached when unresolved refs exist", func(t *testing.T) {
```

Comment:
Maybe something like this
```suggestion
		t.Run("Gateway listener should have AttachedRoutes set even when Gateway has unresolved refs", func(t *testing.T) {
```

Reply:
Commit 8b76391  implements ^ comment.